collect_app/src/main/java/org/odk/collect/android/http/CollectThenSystemContentTypeMapper.java

```diff
@@ -33,21 +33,10 @@ public String map(String fileName) {
     }
 
     private enum CollectContentTypeMappings {
-        XML("xml",  "text/xml"),
```

Comment:
Would you be able to link to what docs/code helped you make the decision to remove these content types?

Reply:
[this one](https://android.googlesource.com/platform/frameworks/base/+/61ae88e/core/java/android/webkit/MimeTypeMap.java)

Reply:
Could you add that reference as a comment to the top of the file. It would be good to have a note that state that "This covers types not included in Android's `MimeTypeMap`" with a link so we know where to look if something goes wrong!

Reply:
Done